dotcom-rendering/index.d.ts

```diff
@@ -323,8 +323,6 @@ type SmallHeadlineSize =
 
 type MediaType = 'Video' | 'Audio' | 'Gallery';
 
-type LineEffectType = 'labs' | 'dotted' | 'straight';
-
```

Comment:
Is this related?

Reply:
this was not being used - sorry to cause extra noise in this PR